test/e2e/tests/network-error.spec.js

```diff
@@ -79,6 +83,9 @@ describe('Gas API fallback', function () {
 
         await driver.clickElement({ text: 'Next', tag: 'button' });
 
+        await driver.delay(regularDelayMs);
```

Comment:
could we possibly avoid using this delay? @DDDDDanica

Reply:
Hey @seaona i have tried to remove but it failed from time to time.  ::(